src/Egret.Cli/Serialization/Audacity/Tag.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Egret.Cli.Serialization.Audacity
+{
+    using System.Xml.Serialization;
+
+    public record Tag
+    {
+        [XmlAttribute(AttributeName = "name")]
+        public string Name { get; init; }
+
+        [XmlAttribute(AttributeName = "value")]
+        public string Value { get; init; }
```

Comment:
~~What does the `Value` attribute do? Just nice to know~~

It'd be good to leave a note that a `Tag` actually acts like a key value pair